servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -429,10 +427,8 @@ public void completeFile(long fileId) throws FileDoesNotExistException, BlockInf
     }
   }
 
-  // TODO(cc) Make it protected instead of public for use in tests, after splitting up
-  // MasterInfoIntegrationTest
-  public void completeFileInternal(List<Long> blockIds, long fileId, long fileLength, long opTimeMs)
-      throws FileDoesNotExistException {
+  protected void completeFileInternal(List<Long> blockIds, long fileId, long fileLength,
```

Comment:
actually, why `protected`? I searched the code base and no one is overriding this method.